deployment/address_book_test.go

```diff
@@ -118,3 +118,47 @@ func TestAddressBook_Merge(t *testing.T) {
 		},
 	})
 }
+
+func TestAddressBook_Remove(t *testing.T) {
```

Comment:
can you add the test for -
should error if all the addresses in other are not contained in the address book.

Reply:
It's covered in the same test https://github.com/smartcontractkit/chainlink/pull/15148/files#diff-bf86f2f08b8765833ff004994d514b9652d6ccf1473a6f921aa8cc39dbd4ea0aR161